charts/rancher-logging/0.2.5/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.3.4-rc1
```

Comment:
I know we haven't updated this lately, but we should bump this to 2.5.10, so that earlier 2.5's stay stable.